### PR TITLE
Fixes #57 - formatter-maven-plugin

### LINKED ISSUE #57: Adopt formatter-maven-plugin or another solution for code formatting
Adopt formatter-maven-plugin or another solution for code formatting

https://code.revelc.net/formatter-maven-plugin/

### PR DESCRIPTION
**Pull request by rsvoboda**
Integration of formatter-maven-plugin and impsort-maven-plugin

Following Quarkus setup and best-practices.

Code gets formatted and imports ordered during the build time (process-sources phase).

Please make sure your PR meets the following requirements:
- [x] Pull Request contains description for the changes
- [x] Pull Request does not include fixes for multiple issues / topics
- [x] Code compiles and tests are passing
- [x] Code is self-descriptive and/or documented

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -155,5 +155,52 @@
...
             </build>
         </profile>
+        <profile>
+            <id>format</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+                <property>
+                    <name>!no-format</name>
+                </property>
+            </activation>
```
Comment (honza-kasik) [inline pom.xml]: Which of these options will take precedence? I understand, that the desired behavior is not to run the check when `no-format` property is defined. Will this override the `activeByDefault` setting? Also, isn't this redundant, when you just can run `mvn clean test -P '!format'` [1]?

[1] http://maven.apache.org/guides/introduction/introduction-to-profiles.html#Deactivating_a_profile

File: ide-config/eclipse.importorder [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#Organize Import Order
```
Comment (honza-kasik) [inline ide-config/eclipse.importorder]: How and where is this file used?

File: README.md [hunk 3]
```diff
@@ -53,4 +53,12 @@ unzip -q eap-microprofile-test-suite-dist.zip
 ```
 Tests output is redirected into `${TEST_FQCN}-output.txt` files in `target/surefire-reports/` directory.
 
-For more details consult with `mp-ts.sh` script
\ No newline at end of file
+For more details consult with `mp-ts.sh` script
+
+## Code format and imports order
```
Comment (honza-kasik) [inline README.md]: IMHO it would be nice to document formatter deactivation here.

File: pom.xml (hunk 1 above)
Comment (rsvoboda) [reply pom.xml]: good point, it was taken from Quarkus, I will update the PR

File: README.md (hunk 3 above)
Comment (rsvoboda) [reply README.md]: Why would we want to promote that use-case ?

File: ide-config/eclipse.importorder (hunk 2 above)
Comment (rsvoboda) [reply ide-config/eclipse.importorder]: It's setting file for Eclipse to be comliat with `impsort-maven-plugin` configuration.

It's just making stuff equivalent between mvn based formatting and Eclipse based.

**Pull request merged**